LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright 2022 Fabulously Optimized Authors
+Copyright 2023 Fabulously Optimized Authors

```

Comment:
`Copyright 2020-2023 Fabulously Optimized Authors`

this makes morely sense in practice.